README.md

```diff
@@ -121,6 +121,7 @@ If you do not have `make`, you can do the following:
 ```sh
 $ ocaml bootstrap.ml
 $ ./boot.exe
+$ ./_boot/default/bin/main.exe build @install
```

Comment:
I think we need the `--release` and `--subst` flags here as well. I general, I'd prefer not to deviate from the commands in the opam file btw. Perhaps we can just copy those and add an install step?

Reply:
It is not necessary to call both `boot.exe` and `main.exe`. Adding `--release` to the `boot.exe` call should be enough.

BTW, the `subst` call is no longer necessary now that we have `dune-build-info`. In particular, the `%%VERSION%%` string no longer appears in the code base, except in a comment.

Reply:
Using [boot --release](https://github.com/Chris00/ocaml-appveyor/commit/3e0ef47181f0278ea685c8484ffdec043ffe0a92#diff-e612dee15b2c34817a46aff4334a02fdR85) does [not work](https://ci.appveyor.com/project/Chris00/ocaml-appveyor/builds/26791368#L5913).

Reply:
Ah, the install command on the following line should take `--build-dir _boot`. The readme wasn't updated when we change the build directory during bootstrap